src/main/java/org/jivesoftware/smackx/pubsub/PubSubIntegrationTest.java

```diff
@@ -915,4 +915,34 @@ public void transientNotificationOnlyNodeWithItemTest() throws NoResponseExcepti
             pubSubManagerOne.deleteNode(nodename);
         }
     }
+
+    /**
+     * Asserts that the server returns an 'item-not-found' error response when
+     * deleting a node that does not exist.
+     *
+     * <p>
+     * From XEP-0060 § 8.4.3.2:
+     * </p>
+     * <blockquote> If the node does not exist, the pubsub service MUST return an
```

Comment:
The javadoc is quoting text from a section of the XEP, but the quoted text is not from the paragraph that is being referred to!